### ISSUE TITLE
[RESULT] Assess the Current Situation

### ISSUE THREAD

**Issue by lazaridiscom** [goerli/parity-goerli#50]
### Task Description
https://github.com/goerli/parity-goerli/issues/46

**Assess the current situation, provide suggestions/alternatives re processing (mostly "Top Level" stuff)**

### Method

"Reviewing Around" within project-resources, slightly within source-code, attempting to understand the system/tasks without asking much (e.g. using only the material within the Product/Project resources).

### Product

Goerli - A PoA-Clique Multi-Client Test-Network for the Ethereum Blockchain

```
PoA      Proof of Authority
Clique   The PoA used by geth (Go Ethereum Client)
Multi    The ability to connect/synchronize via different Client implementation (C++, Go, Rust etc.)
```

### Website

https://goerli.net

* issues: readability, slang/insiderish, no overview
* main action: transform it towards a "Product Website"
  * alternative: create a product website (or at least draft docs)

It is critical that the website informs in a concise manner the different target-audiences, which means that even an "Executive-Summary" style info should be provided.

### Main Project

 * https://github.com/goerli/testnet
 * Github Organisation: https://github.com/goerli
 * issues: missing documentation, missing sys-overview
 * action: create docs, including a system-overview (can be draft level)

### Parity-Clique Project

(main system of focus)

 * https://github.com/goerli/parity-goerli
   * clarification: Parity-Goerli is **not** a Goerli-specific version of Parity-Ethereum Client, but a simple work-fork.

 * Overview of past work: https://github.com/goerli/parity-goerli/pulls?q=is%3Apr+is%3Aclosed
 * Ongoing work: https://github.com/goerli/parity-goerli/branches/active
 * missing: documentation, status, processes

### Summary

There are currently multiple barriers, which make it difficult for external developers (especially newcomers to Eth/Blockchain) to contribute.

## Developer Self-Assessments

(Note to the involved developers: this is not to be seen -in any way- as an attack. Peace!!!)

There are multiple self-assessments of the developers that show that the effort was not estimated correctly:

 * https://github.com/paritytech/parity-ethereum/issues/5687#issuecomment-428844037

Actual assessments indicate that there is still much work (this is a "gut" assessment) :

 * https://github.com/goerli/parity-goerli/issues/44#issuecomment-459205092

"refactoring" does not (by definition) break existent behavior. The fact that this happens indicates again that there is still many work to do:

 * https://github.com/goerli/parity-goerli/issues/44#issuecomment-459211554

All this is nothing special. The main problem could be that

 * once the development is "completed", further integration-issues come up
   * fixing the issues can be difficult...
     * due to the missing modularization of the overall source-codes
     * due to the barriers for new developers to join the efforts (especially for independent, domain-knowledge-free developers, which can operate abstractly on the code)

The remaining implementation/stabilization effort is not clear (not even the tasks are documented fully)

## Alternative/Complementary Process

 * Continue with development as-is (the currently active developers continue without any change and without **any** intervention in their process).
 * In parallel, work on the further above mentioned issues is started (which do not affect the ongoing development)

<hr/>

(what needs to be done next)

EDIT:

A Draft-Specification of the follow-up TASK-B was part of TASK-A. To avoid duplication, deleted it from here. It can be found as a new Issue:

https://github.com/goerli/testnet/issues/40

.

**Comment by 5chdn** [comment]
hey @ceresstation - I cannot payout this bounty

can you help? https://screenshots.firefox.com/jfc1fNB32TI4u5d6/gitcoin.co

hey @lazaridiscom - thanks for your submission. 

I will payout once the issue with Gitcoin is resolved.

**Comment by lazaridiscom** [comment]
> I will payout once the issue with Gitcoin is resolved.

Ok, thank you!

**Comment by 5chdn** [comment]
Paid, thanks"!

**Comment by zero-down2018** [comment]
* clarification: Parity-Goerli is **not** a Goerli-specific version of Parity-Ethereum Client, but a simple work-fork->
thanks for clearing this up, that was not so clear

**Comment by 5chdn** [comment]
Yes, see https://github.com/paritytech/parity-ethereum/pull/9981

**Comment by lazaridiscom** [comment]
@5chdn , "parity-goerli" indicates "a custom version specific for goerli.

The repo-description says currently:

"The Clique Parity working repository "

Possibly just change to

"The Clique Parity working repository (will be merged with original parity-ethereum once stable)"

to make things immediately clear.

**Comment by lazaridiscom** [comment]
The repo now has this description:

"The Clique Parity working repository "

add something like that:

"The Clique Parity working repository (will be merged to the original parity repo)"

thus it becomes immediately clear.
